src/OpenTelemetry/BatchExportProcessorDefaultOptions.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="BatchExportProcessorDefaultOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry
+{
+
+    internal static class BatchExportProcessorDefaultOptions
```

Comment:
Do we need this class, or these consts can be part of the BatchExportProcessor? (e.g. BatchExportProcessor.DefaultMaxQueueSize)?

Reply:
I would move them onto BatchExportProcessorOptions and add "Default" prefix. Primary benefit being discoverability, they will show up on intellisense when user interacts with options instance.

Reply:
I have moved them to BatchExportProcessor as these are the default values to be used for this for BatchExportProcessor ctor parameters. BatchExportProcessorOptions  allows the users to change this value but it has the default values for each of the properties mentioned in the comment. This way it would show up on the property description on Intellisense.